plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -78,6 +96,31 @@
         required: False
         default: False
         type: bool
+    mac_learning:
+        description:
+            - Dictionary which configures MAC learning for portgroup.
+        suboptions:
+            allow_unicast_flooding:
+                type: bool
+                description: The flag to allow flooding of unlearned MAC for ingress traffic.
+                required: False
+            enabled:
+                type: bool
+                description: The flag to indicate if source MAC address learning is allowed.
+                required: False
+            limit:
+                type: int
+                description: The maximum number of MAC addresses that can be learned.
+                required: False
+            limit_policy:
+                type: str
+                description: The maximum number of MAC addresses that can be learned.
```

Comment:
The description is the same as the 113 lines.
Is this correct?

Reply:
Sorry, looks like a stupid copy&paste error from my side. I've updated the description. Thanks!